src/api.rs

```diff
@@ -154,6 +164,9 @@ impl Default for SpeedSettings {
 }
 
 impl SpeedSettings {
+  /// This preset is set to assign the speed, where the speed is passed as arguement,
+  /// the parametes like min_block_size, multi reference preditions, distortion are assigned,
+  /// for instance  *speed_settings: SpeedSettings::from_preset(10);*
```

Comment:
"Set the speed setting according to a numeric speed preset"

Mention which is the range and what's slower/higher quality and what's faster.